cmd/crank/install.go

```diff
@@ -42,6 +43,8 @@ import (
 
 const (
 	errPkgIdentifier = "invalid package image identifier"
+	errKubeConfig    = "failed to get kube-config"
```

Comment:
```suggestion
	errKubeConfig    = "failed to get kubeconfig"
```

Nitpick: can you change all references to `kubeconfig` to be one word? :)

Reply:
Done!

Reply:
@psinghal20 I think you may have missed changing it in this error message